src/main/java/org/opensearch/security/ssl/SecureSSLSettings.java

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Looks like this class is being added, but not utilized. Can you remove this change and/or include an entire backport of https://github.com/opensearch-project/security/pull/2296?

Reply:
I'll create a separate backport for that and remove it from here.